### ISSUE TITLE
Doesn't support sqlite

### ISSUE THREAD

**Issue by boosh** [bitnami/bitnami-docker-laravel#143]
**Description**
If I configure sqlite in the docker compose file it doesn't work:
```
    environment:
      - DB_CONNECTION=sqlite
      - DB_DATABASE=/app/db.sqlite
```
The container just exits. Looking at `app-entrypoint.sh` it's because it always sets up the DB which tries to connect to a database by host and port. 

**Steps to reproduce the issue:**

1. Download the `docker-compose.yml` file
2. Change the env vars to those in the description
3. Run `docker-compose up`

**Describe the results you received:**
Container exits:
`website_myapp_1 exited with code 2`

**Describe the results you expected:**
It would work with sqlite

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**
N/A

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by pablogalegoc** [comment]
Hi @boosh,

I see you've already opened a PR, thank you! A teammate will take a look at it 🙂

**Comment by marcosbc** [comment]
Fix merged in #144
